test/metabase/api/cache_test.clj

```diff
@@ -23,7 +23,7 @@
           (is (mt/user-http-request :crowberto :put 200 "cache/"
```

Comment:
unrelated: is this `is` meant to be here?

Reply:
Yep, this is a kind of `some?` check, saw that approach somewhere in api tests 😁